### PR TITLE
Add event, April 2, 2016

### PR DESCRIPTION
**Pull request by khoeger**
Add event Hip Hop Hacks 2016 (April 2)

### CODE REVIEW

**Review by anilmuppalla — APPROVED** [review]

File: _events/2016-04-02.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+days:
+    - day: 2016-04-02
+      start: 11:30am
+      end: 8:00pm
+    - day: 2016-04-03
+      start: 10:30am
+      end: 6:00pm
+address: "45 W 18th St, New York, NY 10011"
+rsvp: "https://www.eventbrite.com/o/monthly-music-hackathon-nyc-2470452960"
+image: #"place-this-image-file-in-assets-slash-events.jpg"
```
Comment (trevorknight) [inline _events/2016-04-02.md]: Just out of curiosity, I checked. This does mean page.image returns false (which is of course good in this case). 

In general, I'd advocate for just removing anything commented out, so in this case, removing the whole line.

File: _events/2016-04-02.md [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+## Sunday, April 3, 2016
+- 10:30 AM Doors Open
+- 12:00 PM Lunch
+- 3:00 PM Demos of Hacks
+- 5:00 PM Special Guest Performers (to be announced!)
+- 6:00 PM End
+
+# Talks
+
+- **Tony Prendatt Carter**
```
Comment (trevorknight) [inline _events/2016-04-02.md]: Doesn't matter, just FYI and sorry for explaining if you already knew...

Without a new line the descriptions here will be on the same line as the name:
 
**Michael Wiggins** Director of Education for the Urban Arts Partnership

**Review by trevorknight — APPROVED** [review]

File: _events/2016-04-02.md (hunk 1 above)
Comment (khoeger) [reply _events/2016-04-02.md]: Ok! I'll make this change.

File: _events/2016-04-02.md (hunk 2 above)
Comment (khoeger) [reply _events/2016-04-02.md]: I got really strange spacing when I put them on the new lines. I can definitely put it back that way (and get rid of the emphasis.) Want me to?

**Comment by khoeger** [comment]
Have added  the spacing for the presenters, taken away the bold, and commented the image out

**Review by sunnythere — APPROVED** [review]

**Pull request merged**